asv_bench/benchmarks/utils/common.py

```diff
@@ -513,11 +513,11 @@ def execute(
         if ASV_USE_ENGINE == "ray":
             from ray import wait
 
-            all(map(lambda partition: wait([partition.oid]), partitions))
+            all(map(lambda partition: wait([partition.physical_data]), partitions))
```

Comment:
side note - `map()` is an iterator, so using `all()` would stop on _any_ of the `.wait()` returning something false-ish, we should probably use `tuple()` or `list()` if we don't want to use the results of `map()` but want all of its iterations to finish

Reply:
See my comment [here](https://github.com/modin-project/modin/pull/4404#discussion_r877465281): I don't understand why this is different from `[partition.wait() for partition in partitions]`

Reply:
I agree with @vnlitvinov, that implementation looks incorrect because we might not wait of all of partitions computations to finish.

Implementation `[partition.wait() for partition in partitions]` is applicable, but in this case we will stop on the each iteration to wait the each partition. I believe, that something like that will be worked a bit faster:

```python
wait([partition.physical_data for partition in partitions], num_returns=len(partitions))
```

I think, that this kind of changes should be implemented in another PR.

Reply:
@prutskov that's correct. I have filed #4491 for that already.

Reply:
> side note - map() is an iterator, so using all() would stop on any of the .wait() returning something false-ish

@vnlitvinov yes, that's a bug , that's why we wait on `partition.wait() or True` elsewhere, e.g.: https://github.com/modin-project/modin/blob/e67db1dd3ee448eec23a05ffdb3ca38a2b2b709e/asv_bench/benchmarks/utils/common.py#L509